src/stream/src/executor/actor.rs

```diff
@@ -67,15 +70,15 @@ impl ActorContext {
     }
 
     pub fn create_with_metrics(
-        id: ActorId,
-        fragment_id: u32,
+        stream_actor: PbStreamActor,
```

Comment:
```suggestion
        stream_actor: &PbStreamActor,
```
We'd better not clone the actor here, in some cases the `PbStreamActor` could be very large.